Build failed with checkstyle error:
```
[ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk11-scala2.12/streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java:194:26: Variable 'e' should be declared final. [FinalLocalVariable]
```

--- a/streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java
+++ b/streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java
@@ -191,7 +191,7 @@ public class GlobalStateManagerImplTest {
         checkpoint.write(startOffsets);
 
         // initialize will throw exception
-        StreamsException e = assertThrows(StreamsException.class, () -> stateManager.initialize());
+        final StreamsException e = assertThrows(StreamsException.class, () -> stateManager.initialize());
         assertThat(e.getMessage(), equalTo("Encountered a topic-partition not associated with any global state store"));
     }
 

